beacon-chain/core/balances/rewards_penalties.go

```diff
@@ -40,7 +40,7 @@ func ExpectedFFGSource(
 				justifiedAttestingBalance /
 				totalBalance
 	}
-	activeValidatorIndices := helpers.ActiveValidatorIndices(state.ValidatorRegistry, helpers.CurrentEpoch(state))
```

Comment:
please revert all these changes, we use the validator registry in `ActiveValidatorIndices`, this PR has reverted it